package.json

```diff
@@ -87,7 +87,7 @@
     "sugarss": "^2.0.0",
     "svelte": "^3.42.0",
     "ts-jest": "^25.1.0",
-    "typescript": "^4.4.2"
+    "typescript": "^4.5.2"
```

Comment:
Quick question: (when) have we removed the support for v3? 🤔

Reply:
never, these are the dev dependencies, not the peer dependencies.